src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/ProcessedEnvelopeNotifierTest.java

```diff
@@ -58,18 +62,23 @@ void notify_should_send_message_with_right_content() throws Exception {
 
         assertThat(message.getMessageId()).isEqualTo(envelopeId);
         assertThat(message.getContentType()).isEqualTo("application/json");
-
         String messageBodyJson = new String(MessageBodyRetriever.getBinaryData(message.getMessageBody()));
-        String expectedMessageBodyJson = String.format("{\"id\":\"%s\"}", envelopeId);
-        JSONAssert.assertEquals(expectedMessageBodyJson, messageBodyJson, JSONCompareMode.LENIENT);
+        String expectedMessageBodyJson =
+            String.format(
+                "{\"id\":\"%s\",\"processed_ccd_reference\":%s,\"processed_ccd_type\":\"%s\" }",
+                envelopeId,
+                ccdId,
+                envelopeCcdAction
+            );
+        JSONAssert.assertEquals(expectedMessageBodyJson, messageBodyJson, JSONCompareMode.STRICT);
```

Comment:
Is the property order guaranteed (or do we care?)
Maybe it can stay as `LENIENT`?

Reply:
if you say LENIENT when test pass even there are missing fields

Reply:
woah